packages/tools/guiEditor/src/globalState.ts

```diff
@@ -19,6 +20,12 @@ export enum DragOverLocation {
     NONE = 3,
 }
 
+export enum Tool {

```

Comment:
I think "Tool" is a very abstract name for that. Maybe EditorTool, or GUIEditorControlTool, or something more specific?

Reply:
+1

Reply:
Sure, we can do that